hazelcast-jet-core/src/test/java/com/hazelcast/jet/rocksdb/RocksMapTest.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.rocksdb;
+
+import com.hazelcast.internal.serialization.InternalSerializationService;
+import com.hazelcast.jet.JetException;
+import com.hazelcast.jet.core.JetTestSupport;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import static java.util.Collections.emptyMap;
+
+public class RocksMapTest extends JetTestSupport {
+    private RocksDBStateBackend rocksDBStateBackend;
+    private InternalSerializationService serializationService;
+
+    @Before
+    public void init() {
+        serializationService = getJetService(createJetMember())
+                .createSerializationService(emptyMap());
+        rocksDBStateBackend = new RocksDBStateBackend(serializationService);
+    }
+
+    @After
+    public void cleanup() {
+        rocksDBStateBackend.deleteKeyValueStore();
+        serializationService.dispose();
+    }
+
+    @Test
+    public void test_getMap() {
+        RocksMap<String, Integer> rocksMap = rocksDBStateBackend.getMap();
+        Assert.assertNotNull("failed to create RocksMap ", rocksMap);
+    }
+
+    @Test
+    public void test_put_and_get() {
+        //Given
+        RocksMap<String, Integer> rocksMap = rocksDBStateBackend.getMap();
+        //When
+        rocksMap.put("key1", 1);
+        //Then
+        Assert.assertNotNull("the insert wasn't applied", rocksMap.get("key1"));
+        if (rocksMap.get("key1") != 1) {
```

Comment:
```
@Test
public void when_putKeyValue_then_getKeyReturnsValue() {
    // Given
    String key = "key1";
    Integer value = 1;

    // When
    rocksMap.put(key, value);

    // Then
    assertEquals("rocksMap.get() didn't return the value used in rocksMap.put()", value, rocksMap.get(key));
}
```

Note:
- test method naming
- extracted parameters to protect from typos, also explicitly sets the type to `Integer` (relevant for `assertEquals` overload resolution) 
- usage of `assertEquals`, statically imported
- careful wording of error messages
- blank lines between phases for readability